packages/igniter-test-helpers/lib/after-microtasks.ts

```diff
@@ -1,23 +1,36 @@
-import Ember from 'ember';
-import { manager } from 'igniter/microtask-manager';
+/*
+ * WAT. There is some stuff in here that is Ember dependent
```

Comment:
@runspired yeah, to be clear my goal here is 100% to replace backburner in Ember. I don't think we should be concerned about the two world co-existing.